### PR TITLE
LPS-143599 [Impedibug] Info inside project card goes out card's structure when user decreases browser's window size.

### PR DESCRIPTION
**Pull request by agnosmelo**

### CODE REVIEW

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ProjectCard/index.js [hunk 1]
```diff
@@ -14,6 +14,18 @@ const getCurrentEndDate = (currentEndDate) => {
 };
 
 const ProjectCard = ({code, isSmall, onClick, region, sla, status, title}) => {
+	const getStatusMessage = (status) => {
+		if (status === 1) {
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ProjectCard/index.js:18]:
```suggestion
		if (status === status.future) {
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ProjectCard/index.js [hunk 2]
```diff
@@ -14,6 +14,18 @@ const getCurrentEndDate = (currentEndDate) => {
...
+			return 'Ends on  ';
+		}
+
+		if (status === 2) {
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ProjectCard/index.js:22]:
```suggestion
		if (status === status.expired) {
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ProjectCard/index.js [hunk 3]
```diff
@@ -14,6 +14,18 @@ const getCurrentEndDate = (currentEndDate) => {
 };
 
 const ProjectCard = ({code, isSmall, onClick, region, sla, status, title}) => {
+	const getStatusMessage = (status) => {
+		if (status === 1) {
+			return 'Ends on  ';
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ProjectCard/index.js:19]:
```suggestion
			return 'Ends on ';
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ProjectCard/index.js [hunk 4]
```diff
@@ -14,6 +14,18 @@ const getCurrentEndDate = (currentEndDate) => {
...
+		}
+
+		if (status === 2) {
+			return 'Ended on  ';
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ProjectCard/index.js:23]:
```suggestion
			return 'Ended on ';
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ProjectCard/index.js [hunk 5]
```diff
@@ -14,6 +14,18 @@ const getCurrentEndDate = (currentEndDate) => {
...
+		}
+
+		return 'Starts on	 ';
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ProjectCard/index.js:26]:
```suggestion
		return 'Starts on ';
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]
LGTM 🚀, but look my suggestions!

**Comment by jairmedeiros** [comment]
ci:test:cf

**Comment by jairmedeiros** [comment]
ci:test:sf

**Comment by jairmedeiros** [comment]
ci:test:relevant

**Comment by jairmedeiros** [comment]
ci:test:stable

**Comment by agnosmelo** [comment]
ci:test:sf

**Review by kevenleone — APPROVED** [review]
LGTM 🔥

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ProjectCard/index.js [hunk 6]
```diff
@@ -15,15 +15,15 @@ const getCurrentEndDate = (currentEndDate) => {
 
 const ProjectCard = ({code, isSmall, onClick, region, sla, status, title}) => {
 	const getStatusMessage = (status) => {
-		if (status === 1) {
-			return 'Ends on  ';
+		if (status === status.future) {
+			return 'Ends on ';
```
Comment (kevenleone) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ProjectCard/index.js:19]: @jairmedeiros , @agnosmelo let's remove this extra space soon, it's better to implement this need of an extra space, to function caller.

Comment (kevenleone) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ProjectCard/index.js:19]: Just to keep it simple.

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**